app/Locale/French.php

```diff
@@ -209,6 +225,7 @@ public function get_locale(): array
 			'PHOTO_NEW_LICENSE' => 'Ajouter une License',
 			'PHOTO_SET_LICENSE' => 'Sélectionner License',
 			'PHOTO_LICENSE' => 'License',
+			'PHOTO_LICENSE_HELP' => 'Need help choosing?',
```

Comment:
```suggestion
			'PHOTO_LICENSE_HELP' => 'Besoin d\'aide pour choisir ?',
```

Reply:
The strings I didn't apply yet are not added in this PR, they are already in `master`. Should I add them anyway?

Reply:
There are multiple point of view on this.
- One could say it needs to be in a separate PR.
- One could aslo say, given that we are modifying those lines, let's also make sure they are correct. (I am of the later).

@nagmat84 opinion ?

Reply:
Actually, we do not modify the lines. In general, I agree with the later option (and I applied the changes you suggested that really belong to this PR), but the other lines were modified in https://github.com/LycheeOrg/Lychee/commit/4d1b5f0df078bd6160f7815f61d1e9f0447ef6b6, the `consistent_json_api` branch does not contain them and is not up-to-date - if you update this, these lines won't be modified by this PR anymore.

Reply:
@ildyria: Please create another PR for it. I already have been wanting to ask for that, but I did not get to a keyboard fast enough. I am fine with the french translations for `UPLOAD_TRACK` and `DELETE_TRACK` because these strings are added by this PR.

But everything else should have been fixed in the scope of https://github.com/LycheeOrg/Lychee/pull/1270 (which has already been merged) and https://github.com/LycheeOrg/Lychee/pull/1271. As soon as the latter PR and the PR for `consistent_json_api` are merged into master and this PR is re-based these lines vanish anyway.

> One could aslo say, given that we are modifying those lines, let's also make sure they are correct. (I am of the later).

This argument is wrong. We are **not** modifying these lines here.

These lines only show up in this PR as a diff, because this PR is based on `consistent_json_api` and `consistent_json_api` is two or three commits behind current `master`. If the current PRs to `master` are also merged into `consistent_json_api` such that `consistent_json_api` is on par with `master` again, then these lines vanish.